File: Java/src/Tarefa.java
public class Tarefa {
	private long idTarefa;
	private double custo;
	// 0 - Escrita / 1 - Leitura
	private int tipo;
	private int valor;
	
	public Tarefa(long idTarefa, double custo, int tipo, int valor) {
		this.idTarefa = idTarefa;
		this.custo = custo;
		this.tipo = tipo;
		this.valor = valor;
	}

	public long getIdTarefa() {
		return idTarefa;
	}

	public double getCusto() {
		return custo;
	}

	public int getTipo() {
		return tipo;
	}

	public int getValor() {
		return valor;
	}
}

File: GO/src/classes/Tarefa.go
package classes

type Tarefa struct{
	IdTarefa int64
	Custo float64
	Tipo int // 0 - Escrita / 1 - Leitura
	Valor int
}

func CriarTarefa(novoId int64, custo float64, tipo int, valor int) *Tarefa {
	return &Tarefa{
		IdTarefa: novoId, 
		Custo: custo, 
		Tipo: tipo, 
		Valor: valor,
	}
}
